### DISCUSSION TITLE
react-script-editor - not working

### DISCUSSION THREAD

**Discussion by dfernnando** [pnp/sp-dev-fx-webparts#4689] [Q&A]
### Disclaimer

Yes

### Sample

react-script-editor

### Contributor(s)

@wobba @salascz @felixbohnacker

### Question

I am new here! Where do I download the .sppkg file?

**Comment by ghost** [comment]
Thank you for reporting this issue. We will be triaging your incoming issue as soon as possible.

**Comment by wobba** [comment]
It is a sample you need to build yourself. And you should not really use the web part as it poses a security risk by allowing arbitrary script on a page which cannot be governed securely.
